### DISCUSSION TITLE
Further Reading - draft authority

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#48] [General]
Hi Brian,
You have a small paragraph about I-Ds.
IMHO: it makes sense to explain the naming difference between a personal opinion and a group opinion.
draft-<author>-<group>-<name>-<version>
where the "author" may have a special meaning "ietf" - it means the particular group.
Eduard

**Comment by becarpenter** [comment]
It's true, but even draft-ietf-xxx can contain rubbish :-( . I can make a small update.

**Comment by becarpenter** [comment]
Sorry about the delay. Fixed in [4bcc634](https://github.com/becarpenter/book6/commit/4bcc6347759617ee6fa5425b5aff5db922b4b8c5)
